database/migrations/2016_03_13_110032_create_questions_table.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateQuestionsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {   
+        // A question is identified by its id, event's id and the organization's id.
+        Schema::create('questions', function (Blueprint $table)
+        {   
+            $table->increments('id');
+            $table->integer('user_id')->unsigned()->index();
+            $table->foreign('user_id')->references('id')->on('users')
+            ->onDelete('cascade');
+            $table->integer('organization_id')->unsigned()->index();
+            $table->foreign('organization_id')->references('id')->on('organizations')->onDelete('cascade');
+            $table->integer('event_id')->unsigned()->index();
```

Comment:
It is identified in the database(migration) but actually it's not identified in the model(as of the current master branch) as a one-to-many relationship. But in this case (after it is defined as a relationship) then it won't be needed in the questions table.